src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -129,7 +134,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $input->getOption('router'), $env, null)) {
+        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $router, null)) {
```

Comment:
you can avoid explicitly set null to the timeout argument as it's the default value.